fabric-rendering-v1/src/main/java/net/fabricmc/fabric/api/client/rendering/v1/WorldRenderEvents.java

```diff
@@ -213,7 +237,7 @@ private WorldRenderEvents() { }
 	 * down transient state in event handlers or as a hook that precedes hand/held item
 	 * and GUI rendering.
 	 */
-	public static final Event<End> END = EventFactory.createArrayBacked(End.class, callbacks -> context -> {
+	public static final Event<End> END = EventFactory.createArrayBacked(End.class, context -> { }, callbacks -> context -> {
```

Comment:
Is there a reason you are specifying the empty invoker impl for the events?

Reply:
Per the Fabric docs, it makes a small performance difference and these events are called every frame and there are several of them.